package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "apache-daffodil-vscode",
-	"displayName": "Apache Daffodil VS Code Extension",
```

Comment:
`Apache Daffodil VS Code Extension` also shows up in a few other files:
* LICENSE
* NOTICE
* README.md
* build/package/NOLICENSE
* build/package/NONOTICE
* package.json

Should we update it in those files too?

Reply:
Yes we should!  I'll get to those next week.  Thanks.

Reply:
I think I found and replaced all instances of the old name with the new (according to `git grep`).